tests/bluetooth/bsim_bt/bsim_test_notify/src/gatt_server_test.c

```diff
@@ -208,13 +242,63 @@ static void test_main(void)
 	PASS("GATT server passed\n");
 }
 
+static void test_main_multi_notify(void)
+{
+	int err;
+	const struct bt_data ad[] = {
+		BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	};
+
+	err = bt_enable(NULL);
+	if (err != 0) {
+		FAIL("Bluetooth init failed (err %d)\n", err);
+		return;
+	}
+
+	printk("Bluetooth initialized\n");
+
+	err = bt_le_adv_start(BT_LE_ADV_CONN_NAME, ad, ARRAY_SIZE(ad), NULL, 0);
+	if (err != 0) {
+		FAIL("Advertising failed to start (err %d)\n", err);
+		return;
+	}
+
+	printk("Advertising successfully started\n");
+
+	WAIT_FOR_FLAG(flag_is_connected);
+
+	while (bt_eatt_count(g_conn) < CONFIG_BT_EATT_MAX) {
+		k_sleep(K_MSEC(100));
+	}
+	printk("EATT connected\n");
+
+	WAIT_FOR_FLAG(flag_short_subscribe);
+	WAIT_FOR_FLAG(flag_long_subscribe);
+
+	for (int i = 0; i < NOTIFICATION_COUNT / 2; i++) {
+		multiple_notify();
+	}
+
+	while (num_notifications_sent < NOTIFICATION_COUNT / 2) {
```

Comment:
If we assume that `NOTIFICATION_COUNT` is dividable by 2, we should consider adding a BUILD_ASSERT or something to ensure that.

Reply:
BUILD_ASSERT has been already added